app/assets/javascripts/discourse/helpers/category-link.js.es6

```diff
@@ -121,7 +121,7 @@ function defaultCategoryLinkRenderer(category, opts) {
     'data-drop-close="true" class="' +
     classNames +
     '"' +
-    (description ? 'title="' + escapeExpression(description) + '" ' : "") +
+    (description ? 'title="' + description + '" ' : "") +
```

Comment:
Are we 100% sure that `description` is safe?

Reply:
The `description` here is returned by `Category#description_text` that [has been updated in this PR](https://github.com/discourse/discourse/pull/8107/files#diff-af69b723432608c351fb1dd5db9e8348R265) to escape unsafe characters. The return value is cached there, but that's a thread-local cache, so there's no need to worry about stale/unsafe data.

Reply:
I see, kinda confusing... Would have been better to call that variable "`descriptionText`"

Reply:
Agree. I'm changing that.